conans/client/generators/shim.py

```diff
@@ -0,0 +1,32 @@
+import os
+
+from conans.client.shims.write_shims import generate_shim
+from conans.client.tools import get_build_os_arch
+from conans.model import Generator
+
+
+class ShimGenerator(Generator):
```

Comment:
So, could this be removed?

Reply:
Yes, it cannot be implemented following that way without changing the data that generators are using right now.

In the future I'd love to add this as a generator, probably to use together with some of the `virtualenv` family.